pyvista/utilities/fileio.py

```diff
@@ -153,13 +156,22 @@ def read(filename, attrs=None):
         the attribute/method names and values are the arguments passed to those
         calls. If you do not have any attributes to call, pass ``None`` as the
         value.
+    file_format : str, optional
+        Format of file to read with meshio.
 
     """
     filename = os.path.abspath(os.path.expanduser(filename))
     if not os.path.isfile(filename):
         raise IOError('File ({}) not found'.format(filename))
     ext = get_ext(filename)
 
+    # Read file using meshio.read if file_format is present
+    if file_format:
+        try:
+            return read_meshio(filename, file_format)
+        except:
+            pass
```

Comment:
Would `meshio` throw an informative error worth showing the user? if so, this bare except will skip over it

Reply:
> +        try:
+            return read_meshio(filename, file_format)
+        except:
+            pass
Would meshio throw an informative error worth showing the user? if so, this bare except will skip over it

Reply:
Yes, ``meshio`` is throwing some ``AssertionError`` depending on the error. I can remove this ``try/except`` if you think it would be better to show the actual error.

Reply:
Let's take away the try/except - that way if the user passes a non-working `file_format` argument they will be alerted by `meshio`